docs/Makefile

```
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
+SPHINXBUILD   = sphinx-build-3
```

Related to the previous comment 

```suggestion
SPHINXBUILD   = sphinx-build
```

---

As said above, 4 out of the 5 releases I currently test do call it ```sphinx-build-3```, so I think we do want to explicitly call it that way here.